website/source/docs/auth/aws-ec2.html.md

```diff
@@ -423,6 +672,35 @@ The response will be in JSON. For example:
         URL to override the default generated endpoint for making AWS EC2 API calls.
       </li>
     </ul>
+    <ul>
+      <li>
+        <span class="param">iam_endpoint</span>
+        <span class="param-flags">optional</span>
+        URL to override the default generated endpoint for making AWS IAM API calls.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">sts_endpoint</span>
+        <span class="param-flags">optional</span>
+        URL to override the default generated endpoint for making AWS STS API calls.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">iam_server_id_header_value</span>
```

Comment:
We *try* to keep the field documentation in the code in sync with whats in the docs here on the website. Post multiple rounds of review, some of them might have differed. Could you please give a run through the parameter descriptions of all the fields on endpoints and check if the code has the same? If not (like this one here), could you please update them?